ideascube/tests/test_import_users.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+from webtest import Upload
+
+from .factories import UserFactory
+
+pytestmark = pytest.mark.django_db
+User = get_user_model()
+
+
+def test_anonymous_user_should_not_access_import_users_page(app):
+    response = app.get(reverse('user_import'), status=302)
+    assert 'login' in response['Location']
+
+
+def test_normal_user_should_not_access_import_users_page(loggedapp, user):
+    response = loggedapp.get(reverse('user_import'), status=302)
+    assert 'login' in response['Location']
+
+
+def test_staff_should_access_import_users_page(staffapp):
+    assert staffapp.get(reverse('user_import'), status=200)
+
+
+def test_should_create_users(staffapp, monkeypatch):
```

Comment:
Indeed, I missed that. I assumed you were just adding new test functions to test_views.py.